code/API_definitions/qod-provisioning.yaml

```diff
@@ -0,0 +1,1061 @@
+          type: string
+        source:
+          description: Identifies the context in which an event happened in the specific Provider Implementation.
+          type: string
+          format: uri-reference
+        type:
+          description: The type of the event.
+          type: string
+          enum:
+            - "org.camaraproject.qod-provisioning.vwip.status-changed"
+        specversion:
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          type: string
+          enum:
+            - "1.0"
+        datacontenttype:
+          description: 'media-type that describes the event payload encoding, must be "application/json" for CAMARA APIs'
+          type: string
+          enum:
+            - "application/json"
+        data:
+          description: Event notification details payload, which depends on the event type
+          type: object
+        time:
+          description: |
+            Timestamp of when the occurrence happened. It must follow RFC 3339
+          type: string
+          format: date-time
+      discriminator:
+        propertyName: "type"
+        mapping:
+          org.camaraproject.qod-provisioning.vwip.status-changed: "#/components/schemas/EventStatusChanged"
+
+    EventStatusChanged:
+      description: Event to notify a QoD provisioning status change
+      allOf:
+        - $ref: "#/components/schemas/CloudEvent"
+        - type: object
+          properties:
+            data:
+              type: object
+              description: Event details depending on the event type
+              required:
+                - provisioningId
+                - qosStatus
+              properties:
+                provisioningId:
+                  $ref: "#/components/schemas/ProvisioningId"
+                status:
+                  $ref: "#/components/schemas/StatusChanged"
+                statusInfo:
+                  $ref: "#/components/schemas/StatusInfo"
+          required:
+            - data
+
+    StatusInfo:
+      description: |
+        Reason for the new `status`:
+        * `NETWORK_TERMINATED` - Network terminated the QoD provisioning
+        * `DELETE_REQUESTED`- User requested the deletion of the QoD provisioning
+
+      type: string
+      enum:
+        - NETWORK_TERMINATED
+        - DELETE_REQUESTED
+
+    Device:
+      description: |
+        End-user equipment able to connect to the network. Examples of devices include smartphones or IoT sensors/actuators.
+
+        The developer can choose to provide the below specified device identifiers:
+
+        * `ipv4Address`
+        * `ipv6Address`
+        * `phoneNumber`
+        * `networkAccessIdentifier`
+
+        NOTE1: the MNO might support only a subset of these options. The API invoker can provide multiple identifiers to be compatible across different MNOs. In this case the identifiers MUST belong to the same device

```

Comment:
Same change as in https://github.com/camaraproject/QualityOnDemand/pull/326/commits/70ea70ee9386eebd4b7e102887021739cedb1385 tbd (has been changed also in Commonalities)

Reply:
Updated